File: Center Align Excel Cell/Center_Align_Excel_Cell.java
// Create empty Workbook object.
com.aspose.cells.Workbook wb = new com.aspose.cells.Workbook();

// Access first worksheet.
Worksheet ws = wb.getWorksheets().get(0);

// Access cell C4 and add some text inside it.
Cell c4 = ws.getCells().get("C4");
c4.putValue("Aspose File Format APIs");

// Access cell style.
Style st = c4.getStyle();

// Set the horizontal and vertical alignment of the cell style.
st.setHorizontalAlignment(TextAlignmentType.CENTER);
st.setVerticalAlignment(TextAlignmentType.CENTER);

// Set the font color to red.
st.getFont().setColor(Color.getRed());

// Set the cell style.
c4.setStyle(st);

// Set the cell's column width and row height.
ws.getCells().setColumnWidth(c4.getColumn(), 30);
ws.getCells().setRowHeight(c4.getRow(), 40);

// Save the workbook in output Excel format.
wb.save("outputCenterAlignExcelCell.xlsx", SaveFormat.XLSX);

File: Center Align Excel Cell/CenterAlignExcelCell.cs
// Create empty Workbook object.
Aspose.Cells.Workbook wb = new Aspose.Cells.Workbook();

// Access first worksheet.
Worksheet ws = wb.Worksheets[0];

// Access cell C4 and some text inside it.
Cell c4 = ws.Cells["C4"];
c4.PutValue("Aspose File Format APIs");

// Access cell style
Style st = c4.GetStyle();

// Set the horizontal and vertical alignment of the cell style.
st.HorizontalAlignment = TextAlignmentType.Center;
st.VerticalAlignment = TextAlignmentType.Center;

// Set the font color to red.
st.Font.Color = Color.Red;

// Set the cell style
c4.SetStyle(st);

//Set the cell's column width and row height.
ws.Cells.SetColumnWidth(c4.Column, 30);
ws.Cells.SetRowHeight(c4.Row, 40);

// Save the workbook in output Excel format.
wb.Save("outputCenterAlignExcelCell.xlsx", SaveFormat.Xlsx);